dev/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,37 @@
+// For format details, see https://aka.ms/vscode-remote/devcontainer.json or the definition README at
+// https://github.com/microsoft/vscode-dev-containers/tree/master/containers/python-3
+{
+	"name": "Python 3",
+	"context": "..",
+	"dockerFile": "Dockerfile",
+
+	// Use 'settings' to set *default* container specific settings.json values on container create. 
```

Comment:
Does this work? 
https://www.quora.com/How-do-I-write-comments-inside-a-JSON-document

Reply:
yes it is allowed in vscode. I was a bit surprised as well but having some documentation is nice